types/config.ts

```diff
@@ -147,6 +147,9 @@ export const ConnectElementCustomMethodConfig = {
   "financial-account-transactions": {
     setFinancialAccount: (_financialAccount: string): void => {}
   },
+  recipients: {
+    setDataSource: (_dataSource: string): void => {}
```

Comment:
Can we make this a strong type?

Reply:
And also export this type, similar to other exported types in this file (so it can be reused in the react SDK)

Reply:
updated